src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
+/* eslint-disable no-console */
```

Comment:
Consider removing 'eslint-disable no-console' if you intend to have console.log in production code. If it's for debugging purposes only, remember to remove console.log statements before merging the code.